docs/design_decisions.md

```diff
@@ -424,3 +424,24 @@ Dan hoeft de 404 foutcode ook niet gespecificeerd te worden. Als "best practice"
     * 500 Internal Server Error
     * 503 Service Unavailable
     * default
+
+### DD5.24 Gebruik minLength en maxLength voor het afdwingen van de lengte van een waarde
+Indien er voor een waarde een vaste lengte gewenst is en er wordt al gebruik gemaakt van een pattern dan is het logisch om die vaste lengte ook af te dwingen met het pattern.
+We kiezen er echter voor om deze altijd te definiëren met de properties 'minlength' en 'maxLength'.
```

Comment:
"altijd" is wel wat hard gesteld. Er zijn best situaties dat er een reden is het aantal tekens in het pattern op te nemen. Denk aan de postcode (4 cijfers + 2 letters). Ik zou zeggen "indien mogelijk" of "bij voorkeur".

Reply:
Voorstel:

> Overweeg het gebruik van de properties 'minlength' en 'maxLength'. Dit kan voordelen opleveren.

Bij Ratio kunnen we dan je voorbeeld met postcode opnemen als een situatie waarin het geen voordelen oplevert.